.nvmrc

```diff
@@ -1 +0,0 @@
-5
```

Comment:
Good shout, should leave it just so that if we ever put travis on it 👍 it'll know what version to use